daprdocs/content/en/reference/arguments-annotations-overview.md

```diff
@@ -8,7 +8,7 @@ aliases:
   - "/operations/hosting/kubernetes/kubernetes-annotations/"
 ---
 
-This table is meant to help users understand the equivalent options for running Dapr sidecars in different contexts--via the [CLI]({{< ref cli-overview.md >}}) directly, via daprd, or on [Kubernetes]({{< ref kubernetes-overview.md >}}) via annotations.
+This table is meant to help users understand the equivalent options for running Dapr sidecars in different contexts–via the [CLI]({{< ref cli-overview.md >}}) directly, via daprd, or on [Kubernetes]({{< ref kubernetes-overview.md >}}) via annotations.
```

Comment:
```suggestion
This table is meant to help users understand the equivalent options for running Dapr sidecars in different contexts, via:

 - The [CLI]({{< ref cli-overview.md >}}) directly
 - _daprd_
 - [Kubernetes]({{< ref kubernetes-overview.md >}}) annotations
```

Reply:
Here too, I think a list gives less of a sense that it's an "or". Also, it looks very un-balanced visually

Reply:
Ok! I like lists if you can't tell haha. I'm fine either way though :)